tools/c7n_azure/c7n_azure/entry.py

```diff
@@ -10,3 +10,5 @@ def initialize_azure():
     import c7n_azure.policy
     import c7n_azure.container_host.modes
     import c7n_azure.output # noqa
+
+    import c7n_azure.resources.mysql_flexibleserver
```

Comment:
entry imports aren't typically for resources, their more to get modes imported for schema, resources are handled out of the resource_map. ie. this line can be removed.

Reply:
Yes I have removed the line.